src/lib.rs

```diff
@@ -172,6 +215,14 @@ impl KVMi {
         Ok(())
     }
 
+    pub fn control_msr(&self, vcpu: u16, reg: KVMiMsr, enabled: bool) -> Result<(), Error> {
```

Comment:
That may be my fault because I remember suggesting an enum instead of an integer some weeks ago. I don't know though if it had something to do with msrs. @Wenzel You are right that an enum does not make sense for msrs because of their fluctuating nature. @arnabcs17b006 Sorry about that.